satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -128,6 +128,13 @@ model node (
 	field free_bandwidth int64 ( updatable )
 	field free_disk      int64 ( updatable )
 
+ 	field major             int64 ( updatable )
```

Comment:
Why not `int` instead of `int64`. It seems `int` works already pretty well for the `type` field.